### PR TITLE
Loot helper

### PR DESCRIPTION
**Pull request by DevDennis**
- Highlights items above a certain value in the rightclick menu
- Show item quantities in the rightclick menu

image
image

### CODE REVIEW

**Comment by xaviergmail** [comment]
Lovely! However, I would like to propose a few aesthetic changes:

- Make the highlight color use the colors from the grounditems plugin (perhaps move this into grounditems as an additional option?)
- Change the color of the item and not the 'take' option

**Comment by DevDennis** [comment]
colored option
image
vs
colored name
image

**Comment by xaviergmail** [comment]
Lovely, could simply be an option as well to toggle between the two 🎈

File: runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java [hunk 1]
```diff
@@ -70,7 +70,7 @@
 	// The 15 pixel gap between each drawn ground item.
 	private static final int STRING_GAP = 15;
 	// Threshold for highlighting items as blue.
-	private static final int LOW_VALUE = 20_000;
+	static final int LOW_VALUE = 20_000;
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java]: Is there a reason why these are no longer private?

File: runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java [hunk 2]
```diff
@@ -54,4 +80,95 @@ public Overlay getOverlay()
...
+				if (config.highlightMenuItemName())
+				{
+					String target = lastEntry.getTarget().substring(lastEntry.getTarget().indexOf(">") + 1);
+					lastEntry.setTarget(colTag + target);
+				}
+			}
+
+			if (itemComposition.isStackable())
+			{
+				lastEntry.setTarget(lastEntry.getTarget() + " (" + quantity + ")");
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java]: This functionality should have a config option

File: runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java [hunk 3]
```diff
@@ -54,4 +80,95 @@ public Overlay getOverlay()
...
+			return itemManager.getItemPriceAsync(itemComposition.getId());
+		}
+	}
+
+	@Subscribe
+	public void onMenuEntryAdded(MenuEntryAdded event)
+	{
+		if (event.getOption().equals("Take"))
+		{
+			String hiddenItemsStr = config.getHiddenItems().toLowerCase();
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java]: Most of this functionality can be skipped early on depending on the config

File: runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java (hunk 1 above)
Comment (DevDennis) [reply runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java]: Yes those are used in the GroundItemsPlugin class

**Pull request merged**